### ISSUE TITLE
Keep track of 2amigos yii2-usario

### ISSUE THREAD

**Issue by philippfrenzel** [dabilite/yii2-user#23]
Hi,

as 2amigos have put a lot of refactoring into the module, maybe some of the work can be used to keep up and save time for some issues?!

Best, Philipp

**Comment by maxxer** [comment]
Indeed I noticed the same... I wrote in IRC but nobody was around 🤣 

They say it's based on dektrium's one, so it must have inherited all the features. It seems also their module went far ahead from a simple mantinenance and refactoring. For example they implemented *email as username*.

**Comment by thyseus** [comment]
Yeah, the 2amigo guys are really, really cool. Most probably they will replace dektrium in the long term.

**Comment by thyseus** [comment]
Further: as soon as the keep yii2-usuario independent of

https://github.com/2amigos/yii2-app-template
https://github.com/2amigos/yii2-config-kit

i see no relevance in dektrium and/or dabilite anymore. I just hope they don´t integrate it into their applications template because - in my humble opinion - they overburden the yii application a little bit.

**Comment by tonydspaniard** [comment]
@thyseus yii2-usuario is totally independent of yii2-app-template and yii2-config-kit. I used them as an example. 

>  they overburden the yii application a little bit.

Yii2-app-template is our way of doing things. I solved our issues with configuration files with config kit so we could somehow separate the different sections of the configuration array so we could easily find that one that applies to a component or components. I really disliked (on enterprise projects) the fact to go throughout so many unorganized config files or huge config files in order to find the configuration of a certain component. 

Then the proposed architecture is more a nowadays some sort of boilerplate standards (ie based on the research of: https://github.com/php-pds/skeleton). 

Nevertheless, is just an alternative and also an example on how flexible is config-kit to create your very own configuration files set up.  

---- 

yii2-usuario works a bit different (not at the views nor controllers) but I am all up to make it really great. I personally would like it to think further than simply adding the features that exists on Yii2-user and not on yii2-usuario.

**Comment by tonydspaniard** [comment]
re 2amigos/yii2-usuario#12

**Comment by thyseus** [comment]
@tonydspaniard Cool, thanks for your long, elaborating answer. I really see the future in usuario since the original dektrium members did not react for at least half a year. You 2amigos guys have a lot of experience in creating a great module, so we all should put our efforts into usuario now.

I will do the according Pull Requests into yii2-usuario.

**Comment by tonydspaniard** [comment]
@thyseus awesome. I think would be nice to actually write issues first, so we all can decide whether that integration is important or not, and how to implement it. 

btw: I love the idea behind https://github.com/thyseus/yii2-message good work.

**Comment by SamMousa** [comment]
I propose we archive this repository to make it clear for everyone that development has stopped.
@thyseus what do you think?
I will no longer be contributing to this repo since I'm no longer using it and if I need an extension I'll use yii2-usuario.
